doc/source/conf.py

```diff
@@ -183,7 +183,7 @@
 scv_banner_greatest_tag = True
```

Comment:
I think we should replace:

scv_banner_greatest_tag = True

by 

scv_banner_main_ref = "master"